### PR TITLE
Generators 

### PR DESCRIPTION
**Pull request by WebUraXalys**
Creted "genererators.py" with functions "generate_plan_interruption", "generate_emergency_interruption" and "generate_planed_interrupted_buildings", added buttons in Django admin with them functionality.

### CODE REVIEW

File: blackout/blackout/settings.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.getenv('SECRET_KEY')
+SECRET_KEY = 'a'
```
Comment (Prohodchikoff) [inline blackout/blackout/settings.py:15]: SECRET_KEY = [os.getenv('SERVER_URL'), '127.0.0.1']

**Review by Prohodchikoff — CHANGES_REQUESTED** [review]

File: blackout/parserapp/services/generators.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                plan_interruption.save()
+            else:
+                plan_interruption = Interruptions(
+                    Start=datetime(year, month, day, start_hour, 0, 0),
+                    End=datetime(year, month, day, end_hour, 0, 0),
+                    Type="Plan",
+                )
+                plan_interruption.save()
+            break
+
```
[the remark this answers is not in the document]
Comment (Ferkkkk) [reply blackout/parserapp/services/generators.py]: https://github.com/WebUraXalys/blackout/pull/21/commits/a238161dea6333e28bf21ab5d557a69a3497f364

File: blackout/blackout/settings.py (hunk 1 above)
Comment (Ferkkkk) [reply blackout/blackout/settings.py]: https://github.com/WebUraXalys/blackout/pull/21/commits/a238161dea6333e28bf21ab5d557a69a3497f364

**Comment by WebUraXalys** [comment]
Can you describe the API link as an authorized pull request

**Review by Prohodchikoff — CHANGES_REQUESTED** [review]
Fix the issue

**Comment by Ferkkkk** [comment]
> Fix the issue
https://github.com/WebUraXalys/blackout/commit/cbf7e1ab906b690c4a7d2b8efd222db206582708

**Pull request closed without merging**